beeflow/client/core.py

```diff
@@ -410,6 +410,7 @@ def start(foreground: bool = typer.Option(False, '--foreground', '-F',
     if bee_client.get_hostname() == "" and bee_client.check_backend_status() == "":
         bee_client.setup_hostname(start_hn)  # add to client db
     else:
+        time.sleep(5)  # giving Slurm time to relinquish compute node, if necessary
```

Comment:
Please make this 10 seconds, it won't affect most users, thanks!
That works and everything else looks good and works well!
Thanks for helping me find out why it wasn't working for me. The logic seemed fine, I just did not take into consideration slurm's timing.